### PR TITLE
Issue #174 - Remove use of 'digitally-signed'

### PR DESCRIPTION
**Pull request by bifurcation**

### CODE REVIEW

File: draft-ietf-trans-rfc6962-bis.md [hunk 1]
```diff
@@ -755,18 +755,16 @@ which encapsulates a `SignedCertificateTimestampDataV2` structure:
...
     } SignedCertificateTimestampDataV2;
 ~~~~~~~~~~~
 
 `log_id` is this log's unique ID, encoded in an opaque vector as described in
 {{log_id}}.
 
-`timestamp` is equal to the timestamp from the
-`TimestampedCertificateEntryDataV2` structure encapsulated in the
-`timestamped_entry`.
+`timestamp` is equal to the timestamp from underlying the
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: "from the underlying"

File: draft-ietf-trans-rfc6962-bis.md [hunk 2]
```diff
@@ -784,10 +782,13 @@ extension that it does not understand, it SHOULD ignore that extension.
...
-The encoding of the digitally-signed element is defined in [RFC5246].
+`algorithm` indicates the signature scheme used to compute the signature.  The
+SignatureScheme enumeration is defined in {{!I-D.ietf-tls-tls13}}.
 
-`timestamped_entry` is a `TransItem` structure that MUST be of type
-`x509_entry_v2` or `precert_entry_v2` (see {{tree_leaves}}).
+`signature` is the value of a signature computed using the appropriate signature
+algorithm for the log indicated by `log_id`.  The input to the signature is a
+`TransItem` structure that MUST be of type `x509_entry_v2` or `precert_entry_v2`
+(see {{tree_leaves}}).
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Should we say how should the TransItem be encoded prior to being signed?

File: draft-ietf-trans-rfc6962-bis.md [hunk 3]
```diff
@@ -872,7 +872,12 @@ timestamp of the previous update.
 
 `tree_head` contains the latest tree head information (see {{tree_head}}).
 
-`signature` is a signature over the encoded `tree_head` field.
+`algorithm` indicates the signature scheme used to compute the signature.  The
+SignatureScheme enumeration is defined in {{!I-D.ietf-tls-tls13}}.
+
+`signature` is the value of a signature over the encoded `tree_head` field,
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Ditto, for encoding.

**Comment by eranmes** [comment]
@robstradling any opinion on the questions presented?

File: draft-ietf-trans-rfc6962-bis.md [hunk 4]
```diff
@@ -784,10 +782,13 @@ extension that it does not understand, it SHOULD ignore that extension.
 Furthermore, an implementation MAY choose to ignore any extension(s) that it
 does understand.
 
-The encoding of the digitally-signed element is defined in [RFC5246].
+`algorithm` indicates the signature scheme used to compute the signature.  The
+SignatureScheme enumeration is defined in {{!I-D.ietf-tls-tls13}}.
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: For consistency, could we stick to using just explicit references (in the "normative:" and "informative:" sections)?

**Comment by robstradling** [comment]
Closing in favour of PR #260.

**Pull request closed without merging**